lang/en/auth/login.php

```diff
@@ -1,8 +1,9 @@
 <?php
 
 return [
-    'title' => 'Log in test',
-    'email' => 'Email',
+    'title_page' => 'Log in',
+    'title' => 'Log in',
+    'email' => 'Email or phone',
```

Comment:
We will not do phone authentication
```suggestion
    'email' => 'Email',
```